mplc/scenario.py

```diff
@@ -805,22 +837,23 @@ def run(self):
         # -----------------
         # Preliminary steps
         # -----------------
-
-        self.plot_data_distribution()
+        if not self.save_folder is None:
+            self.save_folder.mkdir()
+            self.plot_data_distribution()
         logger.info(f"Now starting running scenario {self.scenario_name}")
 
         # -----------------------------------------------------
         # Instantiate and run the distributed learning approach
         # -----------------------------------------------------
 
-        self.mpl = self.multi_partner_learning_approach(self, is_save_data=True)
+        self.mpl = self._multi_partner_learning_approach(self, custom_name='main_mpl')
         self.mpl.fit()
 
         # ----------------------------------------------------------
-        # Instantiate and run the contributivity measurement methods
+        # Instantiate and run the contributivity measurement contributivity_methods
         # ----------------------------------------------------------
```

Comment:
Lacks the dashes `...---...` to make it pretty 😃

Reply:
sorry, I didn't understand

Reply:
The dashets `------` should match the entire length of the comment line